src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -208,3 +215,20 @@ function transformExpectArg(arg: any): unknown {
 
     return arg;
 }
+
+async function getWdioInstance(
+    session: WebdriverIO.Browser,
+    element?: WebdriverIO.Element,
+): Promise<WebdriverIO.Browser | WebdriverIO.Element> {
+    const wdioInstance = element ? await session.$(element) : session;
+
+    if (isWdioElement(wdioInstance) && !wdioInstance.selector) {
+        wdioInstance.selector = element?.selector as Selector;
```

Comment:
I need recover selector because it is undefined when element found by `elementId`